style.css

```diff
@@ -64,3 +66,28 @@ nav.links > a:not(last-child) {
 nav.links > a:hover {
   text-decoration: none;
 }
+
+nav.links > div.nav-item > a {
+  color: #000;
+  font-size: calc(8px + 1vh);
+  padding-left: 8px;
+}
+
+nav.links > div.nav-item > a:not(last-child) {
+  margin-bottom: 2vh;
+}
+
+nav.links > div.nav-item > a:hover {
+  text-decoration: none;
+}
+
+nav.links > div.nav-item > img.img-content {
+  width: 24px;
```

Comment:
We made it so the spaces and font-sizes on this projects vary according to the viewport size. IMO we should make the same for icon sizes so they don't change proportion relative to the text on different screen dimensions.

Reply:
fixed :smiley: